### DISCUSSION TITLE
Prevent VerifyChecks.Run from failing when also using DerivePathInfo

### DISCUSSION THREAD

**Discussion by slimCODE** [VerifyTests/Verify#1357] [Q&A]
For a special scenario, we have a library assembly implementing the base code that actually calls `Verify`, and unit test projects derive from that code. The default behavior causes verified files to be created in the library's project instead of each unit test project. To get around this, each unit test project calls `DerivePathInfo` like this:
```
DerivePathInfo((sourceFile, projectDirectory, type, method) => new(projectDirectory));
```
The side effect is that `VerifyChecks.Run` now fails with error "Found incorrectly imported verified file.". Is there any way to skip or adjust the directory expectation of that check?

**Comment by SimonCropp** [comment]
can u share a stand-alone repro of this issue?

**Reply by slimCODE** [reply]
I found the issue and it's not about `DerivePathInfo` at all, my bad. It's because the unit tests were moved from one project to another, and after having addressed `VerifyChecks`'s legitimate "verified files in wrong projects" issues, I thought I was stuck with the same error. Should have read better, the csproj had invalid references to the verified files.

**Reply by SimonCropp** [reply]
@slimCODE no worries. glad u worked out the problem
